src/shredder.rs

```diff
@@ -254,9 +254,9 @@ pub trait Shredder {
 pub struct RegularShredder;
 
 impl Shredder for RegularShredder {
-    const MAX_DATA_SIZE: usize = MAX_DATA_PER_SLICE;
-    const DATA_OUTPUT_SHREDS: usize = DATA_SHREDS;
     const CODING_OUTPUT_SHREDS: usize = TOTAL_SHREDS - DATA_SHREDS;
+    const DATA_OUTPUT_SHREDS: usize = DATA_SHREDS;
+    const MAX_DATA_SIZE: usize = MAX_DATA_PER_SLICE;
```

Comment:
I liked the previous order way better, especially CODING before DATA, is very confusing, we do it the other way around everywhere else